nemo_rl/experience/rollouts.py

```
@@ -893,3 +902,62 @@ async def run_single_sample_with_error_handling(i, sample_state):
         return final_batch, rollout_metrics
 
     return asyncio.run(_async_rollout_implementation())
+
+def _append_next_user_question(
+    batch: "BatchedDataDict",
+    idx: int,
+    tokenizer: "AutoTokenizer",
```

Ruff (0.13.1)

Undefined name `AutoTokenizer`

(F821)